apps/bttester/syscfg.yml

```diff
@@ -97,6 +97,8 @@ syscfg.vals:
     BLE_ISO_MAX_BISES: 1
     BLE_ISO_MAX_BIGS: 1
     BLE_EXT_ADV: 1
+    BLE_PHY_2M: 1
+    BLE_EXT_ADV_MAX_SIZE: 261
```

Comment:
why is this needed? it is quite big value

Reply:
Updated the value to `40` - previous value was an overkill. This is necessary for broadcast, and specially extended advertising configuration in broadcast bttester application.